packages/babel-plugin-transform-typescript/src/index.js

```diff
@@ -205,11 +205,19 @@ export default function() {
       },
 
       TSImportEqualsDeclaration(path) {
-        throw path.buildCodeFrameError("`import =` is not supported.");
+        throw path.buildCodeFrameError(
+          "`import =` is not supported by @babel/plugin-transform-typescript \n" +
```

Comment:
You can remove the trailing space

Reply:
sure.

Reply:
@nicolo-ribaudo it was there earlier too. If I remove it on terminal the words look very closely attached. 
![screen shot 2017-12-21 at 4 11 16 am](https://user-images.githubusercontent.com/5646166/34232148-08504876-e605-11e7-9f6f-4e136ffb10ec.png)

Reply:
Oh, I meant before `\n`

Reply:
Done.